plugins/generator-1.18.1/forge-1.18.1/procedures/world_data_temperature.java.ftl

```diff
@@ -0,0 +1 @@
+/*@float*/(world.getBiome(new BlockPos((int) ${input$x}, (int) ${input$y}, (int) ${input$z})).getBaseTemperature())
```

Comment:
Verify how this was before and now, before it was multiplied by 100, but not sure what units new method has. Please test, match values and report findings. Thanks!